src/main/java/picard/util/LiftoverUtils.java

```diff
@@ -89,7 +84,16 @@ public static VariantContext liftVariant(final VariantContext source, final Inte
 
         builder.filters(source.getFilters());
         builder.log10PError(source.getLog10PError());
-        builder.attributes(source.getAttributes());
+
+        // If any of the source alleles is symbolic, do not populate the END tag (protecting things like <NON_REF> from
+        // getting screwed up in reverse-complemented variants
+        if (source.hasAttribute(VCFConstants.END_KEY) && builder.getAlleles().stream().noneMatch(Allele::isSymbolic)) {
```

Comment:
Should there even be an `END` attribute if there is no symbolic allele? (There probably is a reason to have it I just don't know it)

from the VCFv4.3 spec
```
END 1 Integer End position (for use with symbolic alleles)
```

Reply:
if you look at the original issue that spawned this PR you'll see an example of an insertion with an END allele.

Reply:
but shouldn't we be getting rid of `END` in that case?

Reply:
hmmm. not sure. if someone has an "end" I don't mind moving it for them....but not in the case of an inversion (where I do remove it)

Reply:
I'm 50/50 on what should happen here but I'm happy leaving it up to you